test/e2e/util.go

```diff
@@ -366,12 +366,21 @@ func createTestingNS(baseName string, c *client.Client) (*api.Namespace, error)
 		},
 		Status: api.NamespaceStatus{},
 	}
-	got, err := c.Namespaces().Create(namespaceObj)
+	// Be robust about making the namespace creation call. We can't
+	// use wait.Poll() because we want to return a value (the namespace).
+	var got *api.Namespace
+	var err error
+	for t := time.Now(); time.Since(t) < singleCallTimeout; time.Sleep(5 * time.Second) {
```

Comment:
- @quinton-hoole will probably tell you to use `wait.Poll(...)` here instead of a custom method
- and I would probably tell you to use `poll` instead of `5 * time.Second`